### ISSUE TITLE
onPushAccepted:withNotification:onStart: delegate method not called on 4.1.2

### ISSUE THREAD

**Issue by Antowkos** [Pushwoosh/pushwoosh-ios-sdk#44]
Hi! I was using version `4.0.7` this way:

```
[PushNotificationManager initializeWithAppCode:appCode appName:appName];

PushNotificationManager *pushManager = [PushNotificationManager pushManager];
[pushManager setShowPushnotificationAlert:NO];
[pushManager setDelegate:self];
[pushManager handlePushReceived:launchOptions];
[pushManager sendAppOpen];
[pushManager registerForPushNotifications];
```

and everything was working fine, i mean delegate method `onPushAccepted:withNotification:onStart:` called after using method `handlePushReceived:`.

But when I did update sdk to version `4.1.2` delegate method `onPushAccepted:withNotification:onStart:` stopped calling. Could you check it, please?

**Comment by DimanAM** [comment]
Did it stop getting called at all or only for launch notifications? What launchOptions are passed into
```objective-c
[pushManager handlePushReceived:launchOptions];
```
Can you print them?

**Comment by Antowkos** [comment]
Yes, it stop getting called at all. It doesn't matter what i will pass there. As i already said on version `4.0.7` everything works fine.

**Comment by DimanAM** [comment]
This is probably connected with missing hash ("p") in notification payload. SDK stopped handling such notifications since `4.1.0`. This problem should be fixed in `4.1.5`.

**Comment by bejayoharen** [comment]
@DimanAM We have a live app with this problem running `4.1.2`. Is it possible to modify our pushes we send so push manager on iOS works, or is the only solution to ensure that clients are updated to `4.1.5`? I don't understand what the "missing hash ("p") in notification payload" is.

**Comment by viridanti** [comment]
Fixed in 4.1.5
